test/integration-serial/serial_deploy_test.rb

```diff
@@ -161,6 +161,24 @@ def test_crd_pruning
     wait_for_all_crd_deletion
   end
 
+  def test_custom_resources_predeployed
+    assert_deploy_success(deploy_fixtures("crd-predeploy", subset: %w(mail.yml things.yml widgets.yml)))
```

Comment:
I'd prefer if you used the fixtures in `test/fixtures/crd` and added the annotations in the test, rather than create a new directory.

Reply:
Done.